xontrib/voxapi.py

```diff
@@ -114,14 +119,23 @@ def __init__(self):
         else:
             self.venvdir = builtins.__xonsh__.env['VIRTUALENV_HOME']
 
-    def create(self, name, *, system_site_packages=False, symlinks=False,
-               with_pip=True):
+    def create(
+        self,
+        name,
+        *,
+        interpreter=VOX_DEFAULT_INTERPRETER,
```

Comment:
I would maybe change this to `interpreter=None` and then in the body of the function, have 

```
interpreter = builtins.__xonsh__.env.get("VOX_DEFAULT_INTERPRETER", sys.executable) if interpreter is None else interpreter
```